Client/game_sa/CModelInfoSA.h

```diff
@@ -36,7 +36,9 @@ class CPedModelInfoSAInterface;
 #define     ARRAY_ModelInfo                 0xA9B0C8
 #define     CLASS_CText                     0xC1B340
 #define     FUNC_CText_Get                  0x6A0050
-#define     FUNC_GetModelFlags              0x4044E0
+#define     FUNC_SetBaseModelFlags          0x5B3AD0
+#define     FUNC_SetAtomicModelFlags        0x5B3B20
+#define     FUNC_GetModelStreamFlags        0x4044E0
```

Comment:
IIRC we've stopped using defines for function addresses.
@saml1er am I correct?

Reply:
Hmm... Unused